config/default.yml

```diff
@@ -335,6 +335,11 @@ Capybara/FeatureMethods:
   Enabled: true
   StyleGuide: http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Capybara/FeatureMethods
 
+FactoryGirl/DynamicAssociationDeclaration:
+  Description: Prefer declaring associations using `#association`.
```

Comment:
you have to sync this with the first comment in the cop itself. Run `bin/build_config` - it will do the job